src/v/raft/types.h

```diff
@@ -613,6 +616,9 @@ struct install_snapshot_reply
     // indicates if the request was successfull
     bool success = false;
 
+    // replying node
```

Comment:
nit: update << operator output to include this?

Reply:
oh, yeah, i will update this PR with a test and auto reset of the sequence number, just in case